src/router/routes_conf.ts

```diff
@@ -11,109 +12,127 @@ const routesConf: RoutesConf = {
   community: {
     path: '/community',
     children: {
+      fullPath: '/community',
       name: 'community',
     },
   },
   company_coordinator: {
     path: '/company-coordinator',
     children: {
+      fullPath: '/company-coordinator',
       name: 'company-coordinator',
     },
   },
   home: {
     path: '/',
     children: {
+      fullPath: '/',
       name: 'home',
     },
   },
   login: {
     path: '/login',
     children: {
+      fullPath: '/login',
       name: 'login',
     },
   },
   prizes: {
     path: '/prizes',
     children: {
+      fullPath: '/prizes',
       name: 'prizes',
     },
   },
   register: {
     path: '/register',
     children: {
+      fullPath: '/register',
       name: 'register',
     },
   },
-  'register-coordinator': {
+  register_coordinator: {
     path: '/register-coordinator',
     children: {
+      fullPath: '/register-coordinator',
       name: 'register-coordinator',
     },
   },
-  'register-challenge': {
+  register_challenge: {
     path: '/register-challenge',
     children: {
+      fullPath: '/register-challenge',
       name: 'register-challenge',
     },
   },
   results: {
     path: '/results',
     children: {
+      fullPath: '/results',
       name: 'results',
     },
   },
   results_detail: {
-    path: '/results/detail',
+    path: 'detail',
     children: {
-      name: 'results/detail',
+      fullPath: '/results/detail',
+      name: 'results_detail',
```

Comment:
Use same hyphen symbol for concatenate string in name key value across code, please.

Example:

`name: 'register-coordinator'`

`name: 'results-detail'` not `name: 'results_detail'`

Reply:
Updated.